pkg/common/ntp.toml.default

```diff
@@ -0,0 +1,50 @@
+# You can configure ntpd-rs with different output levels of logging information
+# Basic values for this are `trace`, `debug`, `info`, `warn`, `error` and `off`.
+# However it is possible to specify more advanced log filters using the
+log-filter = "info"
+
+# Allowed values: full, compact, pretty, json
+log-format = "full"
+
+## The peers section allows configuring peers, you may configure multiple of
+## these blocks to add more peers to your configuration.
+## Our default configuration spawns a pool of peers (by default this attempts
+## to discover 4 distinct peers).
+[[peers]]
+mode = "pool"
+addr = "ntpd-rs.pool.ntp.org"
+#addr = "2.ntpd-rs.pool.ntp.org" # For IPv6 only networks
+max-peers = 4
+
+## If you have an NTS server, you can configure a peer that connects using NTS
+## by adding a configuration such as the one below
+#[[peers]]
+#mode = "nts-server"
+#addr = "localhost:123"
+#ke-addr = "localhost:4460"
```

Comment:
Suggest we not make this localhost, but instead a public server such as the ones from netnod (Note: request for permission in progress)
```suggestion
# NTS service from NETNOD: https://www.netnod.se/nts/network-time-security
#ke-addr = "nts.netnod.se"
```